packs/feats/current-spell.json

```diff
@@ -63,6 +70,7 @@
         "traits": {
             "rarity": "common",
             "value": [
+                "archetype",
```

Comment:
Not sure about adding the trait actually. Need to figure out how we want to handle these I think.

Reply:
Want me to remove `archetype` trait?

Reply:
PR updated - you should now see the two feats removed and the updated Journal Entry.